test/regress/cli/CMakeLists.txt

```diff
@@ -473,14 +475,50 @@ set(regress_0_tests
   regress0/bv/holes/extract-whole.smt2
   regress0/bv/holes/mult-distrib-const-neg.smt2
   regress0/bv/holes/not-neq.smt2
+  regress0/bv/holes/repeat-eliminate-1.smt2 # Calculation may be too complicated
   regress0/bv/holes/repeat-eliminate-2.smt2
   regress0/bv/holes/rotate-left-eliminate-1.smt2
   regress0/bv/holes/sgt-eliminate.smt2
   regress0/bv/holes/shl-by-const-1.smt2
   regress0/bv/holes/udiv-zero.smt2
+  regress0/bv/holes/uge-eliminate.smt2
   regress0/bv/holes/ugt-eliminate.smt2
   regress0/bv/holes/ult-one.smt2
+  regress0/bv/holes/sdiv-eliminate-fewer-bitwise-ops.smt2 # Timeout!
   regress0/bv/holes/zero-extend-eliminate.smt2
+  regress0/bv/holes/sign-extend-eliminate.smt2 # Roundabout method used
```

Comment:
Should this be included if it does not test the rule it was generated to test? Maybe in the hopes that it will use it eventually with progress in the reconstructor?

Reply:
The hope is that eventually it would test `bv-sign-extend-eliminate`. Now it does not, which is a problem.